### ISSUE TITLE
Feat: Consider releasing convenient binaries for all supported platforms

### ISSUE THREAD

**Issue by aalmiray** [project-stacker/stacker#616]
### Is your feature request related to a problem? Please describe.

It's not user friendly asking consumers to build their own binaries (although some prefer to always build from source).

### Describe the solution you'd like

Consider configuring GoReleaser https://goreleaser.com/ for building and releases binaries for all supported platforms

**Comment by mikemccracken** [comment]
Hi @aalmiray - thanks for writing. 
We do ship a statically linked binary for linux, which is the only currently supported platform. 
See the https://stackerbuild.io/get_started/get_stacker/ page for info on downloading and checking system requirements.

Is there a specific other platform you are interested in, or was the download link not prominent enough on the main page?

Thanks,
mike

**Comment by aalmiray** [comment]
Yes, I saw the binary as a release asset but given that there's no explicit indication of the target platform (i.e, linux-x86_64).

Also, being a Go based project it could potentially run on any platform unless the codebase explicitly requires Linux support, wouldn't you agree?

**Comment by mikemccracken** [comment]
Yes, good point that we don't make a note of the arch - that said,
Stacker does require linux - it statically links liblxc and depends on overlayfs, which is a linux filesystem.
It's possible to imagine a stacker that worked on other platforms, but it would be a major change and is not in the scope of the project.
